dev-aws/account-platform/topics.tf

```diff
@@ -94,6 +94,7 @@ resource "kafka_topic" "account_events_v3" {
 resource "kafka_topic" "account_exceptions_events" {
   config = {
     "compression.type" = "snappy"
+    "cleanup.policy"   = "compact"
```

Comment:
No need for more compaction settings like retention time etc?

Reply:
Not unless we want to change from defaults.